packages/firebase_ml_vision/test/firebase_ml_vision_test.dart

```diff
@@ -795,7 +795,7 @@ void main() {
                 'languageCode': 'cd',
               }
             ],
-            'confidence': 0.1,
+            'confidence': 0,
```

Comment:
Should we still have tests of confidence being `0.1`?

Reply:
Validity of confidence was checked multiple times with various non-corner case values.
However, it is probably better to add new test cases instead of modifying old ones to avoid introducing more errors by accident. 

Confidence here is now changed to the original value.

Reply:
Thanks for updating, feel free to remove tests that you believe are unnecessary or redundant in a follow up PR.